test/framework/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch;
+
+import java.util.List;
+
+/**
+ * Provides access to all known transport versions.
+ */
+public class TransportVersions {
```

Comment:
I think this name is too close to TransportVersion. Maybe `AllTransportVersions`?

Reply:
Changed to KnownTransportVersions